jest-setup.js

```diff
@@ -17,16 +17,42 @@ jest.mock('react-native-reanimated', () => {
 });
 global.__reanimatedWorkletInit = jest.fn();
 jest.mock('react-native-gesture-handler',
-  () => ({
-    FlatList: require('react-native').FlatList
-  }),
+  () => {
+    jest.requireActual('react-native-gesture-handler/jestSetup');
+    const GestureHandler = jest.requireActual('react-native-gesture-handler');
+    GestureHandler.Gesture.Pan = () => {
+      const PanMock = {
+        _handlers: {}
+      };
+
+      const getDefaultMockedHandler = handlerName => handler => {
+        if (typeof handler === 'function') {
+          PanMock._handlers[handlerName] = handler;
+        }
+        return PanMock;
+      };
+
+      PanMock.onStart = getDefaultMockedHandler('onStart');
+      PanMock.onUpdate = getDefaultMockedHandler('onUpdate');
+      PanMock.onEnd = getDefaultMockedHandler('onEnd');
+      PanMock.prepare = jest.fn();
+      return PanMock;
+    };
+
+    return GestureHandler;
+  },
   {virtual: true});
 jest.mock('@react-native-picker/picker', () => ({Picker: {Item: {}}}));
 jest.mock('react-native', () => {
   const reactNative = jest.requireActual('react-native');
   reactNative.NativeModules.KeyboardTrackingViewTempManager = {};
   return reactNative;
 });
+jest.mock('./src/incubator/Dialog', () => {
+  const Dialog = jest.requireActual('./src/incubator/Dialog');
+  Dialog.default._inTest = true;
```

Comment:
Is it me or you don't use this after all?

Reply:
Used [here](https://github.com/wix/react-native-ui-lib/pull/2441/files#diff-2692883c2610fa29ab975894c726a1d04f3c41c3f64a2d0c7ded257267600efbR19)

Reply:
Right missed it.